src/frontend/pages/blogs/edit-a-blog/EditBlog.js

```diff
@@ -0,0 +1,317 @@
+import React, { useMemo } from 'react';
+import ImagePreview from '../../../components/ImagePreview/index';
+import FormContainer from '../../../components/FormContainer/index';
+import Button from '../../../components/Button/index';
+import TextInput from '../../../components/TextInput/index';
+import useBlogForm from '../../../hooks/blog-form'
+import styled from 'styled-components';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import CloseIcon from '@material-ui/icons/Close';
+import IconButton from '@material-ui/core/IconButton';
+import UnstyledSelector from '../../../components/Selector';
+
+const EditBlogPage = styled.div`
+  padding: 64px 88px 65px 58px;
+`;
+
+const TopRow = styled.div`
+  display: flex;
+`;
+
+const Selector = styled(UnstyledSelector)`
+  max-width: 250px;
+  width: 100%;
+`;
+
+const EditBlogBody = styled.div`
+  padding-left: 30px;
+`;
+
+const ImagesText = styled.div`
+  font-family: IBM Plex Sans;
+  font-size: 18px;
+  color: #232535;
```

Comment:
I think you defined these fonts inside the `theme.js` as `lights` right ?

Is it possible to use these values defined in theme here ?